### PR TITLE
Vehicle Racks merge and fixes

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Merge changes in master (4NEC2, etc)
- Additional merge fixes
- Power fixes.
- Fixes duplicate entries in ace_interaction menu

### CODE REVIEW

File: addons/sys_server/fnc_getRadioId.sqf [hunk 1]
```diff
@@ -21,10 +21,10 @@ _class = toLower _class;
 
 GVAR(radioIdMap) params ["_baseTypes","_radioIds"];
 
-private _mapIndex = _baseTypes find (toLower _class);
```
Comment (jonpas) [inline addons/sys_server/fnc_getRadioId.sqf:24]: Are you sure? `find` is case-sensitive.

Comment (TheMagnetar) [reply addons/sys_server/fnc_getRadioId.sqf:24]: Yes, If you look at the beginning of the file, class is already set to lower cases.

Comment (jonpas) [reply addons/sys_server/fnc_getRadioId.sqf:24]: Ah I see. 👍

File: addons/sys_rack/fnc_mountRadio.sqf [hunk 2]
```diff
@@ -18,11 +18,11 @@
...
-    [_unit, _radioId] call EFUNC(core,removeGear);
+    [_unit, _radioId] call EFUNC(sys_core,removeGear);
 };
 
 //Stash Radio
 SET_STATE_RACK(_rackId,"mountedRadio",_radioId);
 
 //Send event to do rack specific actions.
-[_rackId, "mountRadio", _radioId] call EFUNC(sys_data,dataEvent);
+[_rackId, "mountRadio", _radioId] call EFUNC(sys_data,dataEvent);
```
Comment (Soldia1138) [inline addons/sys_rack/fnc_mountRadio.sqf]: No newline at EOF

File: addons/sys_rack/fnc_stopUsingMountedRadio.sqf [hunk 3]
```diff
@@ -35,4 +35,4 @@ if (ACRE_ACTIVE_RADIO == _radioId) then { // If it is the active radio.
     // Switch active Radio
     ACRE_ACTIVE_RADIO = "";
     [1] call EFUNC(sys_list,cycleRadios); // Change active radio
-};
+};
```
Comment (Soldia1138) [inline addons/sys_rack/fnc_stopUsingMountedRadio.sqf]: No newline at EOF

**Comment by TheMagnetar** [comment]
This should be ready to go. Thanks @Soldia1138 !

**Pull request merged**